engines/wintermute/ad/ad_scene.cpp

```diff
@@ -1933,7 +1934,16 @@ bool AdScene::scCallMethod(ScScript *script, ScStack *stack, ScStack *thisStack,
 	//////////////////////////////////////////////////////////////////////////
 	else if (strcmp(name, "EnableLight") == 0) {
 		stack->correctParams(1);
-		stack->pushBool(false);
+
+		const char *name = stack->pop()->getString();
```

Comment:
Beware that these create variable shadowing-warnings. Should probably be fixed (but can be handled after this PR, as they are very visible in the compiler log, and thus not likely to be missed)